debian_server/privacyidea-apache2.postinst

```diff
@@ -124,8 +124,10 @@ create_gpg() {
 
 
 set_path() {
-	# Allow pi-manage to be called from everywhere
-	ln -s /opt/privacyidea/bin/pi-manage /usr/local/bin
+  if [ -z "$(grep '/opt/privacyidea/bin' /etc/environment)" ]; then
```

Comment:
Sorry if I'm being overly pedantic, but would it be cleaner to add a file ``/etc/profile/privacyidea.sh`` which has 
```
export PATH=${PATH}:/opt/privacyidea/bin
```
?
(Java does it like this on my system)

Reply:
This is an error anyway! and should be deprecated. We are using links, now!

Reply:
Oh damn. What is this PR? It is WRONG! It removes the paths.

Reply:
I close this PR. And issue a new one!